src/main/java/duke/task/Storage.java

```diff
@@ -0,0 +1,80 @@
+package duke.task;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+/**
+ * This class implements a storage object that deals with loading tasks from the file and saving tasks in the file.
+ *
+ * CS2103T ip
+ * AY21/22 Semester 1
+ *
+ * @author Kishendran Vendar Kon (Group G05)
+ */
+public class Storage {
+    private File f;
```

Comment:
Maybe the naming should be more descriptive? 
It is an attribute that would be used in the entire class, so maybe making it more understandable is a good idea?